packages/os/driverdog.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Load additional drivers
```

Is this more accurate?

```suggestion
Description=Link and load additional drivers
```